src/main/java/nl/tudelft/scrumbledore/game/ScoreCounter.java

```diff
@@ -59,17 +58,24 @@ public void updateScore(int addScore) {
     score += addScore;
 
     if (Constants.isLoggingWantPoints()) {
-      Logger.getInstance()
-          .log("Player gained " + addScore + " points, totalling at " + score + " points.");
+      Logger.getInstance().log(
+          "Player gained " + addScore + " points, totalling at " + score + " points.");
     }
 
     if (score > highScore) {
       highScore = score;
 
       if (Constants.isLoggingWantPoints()) {
-        Logger.getInstance()
-            .log("The high-score has been changed and is now worth " + highScore + " points!");
+        Logger.getInstance().log(
+            "The high-score has been changed and is now worth " + highScore + " points!");
```

Comment:
Maybe only update the highscore when the player dies, instead of at every score increase after exceeding the previous highscore. But let's discuss that in a future sprint.